docker/test/integration/resources/minifi/minifi-log.properties

```diff
@@ -2,3 +2,4 @@ spdlog.pattern=[%Y-%m-%d %H:%M:%S.%e] [%n] [%l] %v
 appender.stderr=stderr
 logger.root=TRACE,stderr
 logger.org::apache::nifi::minifi=TRACE,stderr
+logger.org::apache::nifi::minifi::SchedulingAgent=INFO,stderr
```

Comment:
missing newline at the end of file?

Reply:
good catch, I've reverted the changes to this file though https://github.com/apache/nifi-minifi-cpp/pull/1574/commits/b1355bf745e30cdec6973541b6249e55b368da08, since it will be fixed in MINIFICPP-2125